libmscore/mscore.cpp

```diff
@@ -130,7 +130,7 @@ std::vector<MScoreError> MScore::errorList {
       { NO_CHORD_REST_SELECTED,          "s2", QT_TRANSLATE_NOOP("error", "No chord/rest selected:\nPlease select a chord or rest and retry")      },
       { NO_LYRICS_SELECTED,              "s3", QT_TRANSLATE_NOOP("error", "No note or lyrics selected:\nPlease select a note or lyrics and retry") },
       { NO_NOTE_REST_SELECTED,           "s4", QT_TRANSLATE_NOOP("error", "No note or rest selected:\nPlease select a note or rest and retry")     },
-      { NO_NOTE_SLUR_SELECTED,           "s5", QT_TRANSLATE_NOOP("error", "No note or slur selected:\nPlease select a note or slur and retry")     },
+      { NO_FLIPPABLE_SELECTED,           "s5", QT_TRANSLATE_NOOP("error", "No flippable element selected:\nPlease select a flippable element and retry") },
```

Comment:
Shure it gets written with a double 'p'?
Now to the problem how to translate this... Guess I'd have to use 'umdrehbares Element' for German.

Reply:
See https://en.wiktionary.org/wiki/flippable.
Don't know German ;-)

Reply:
Yes, "flippable" is correct.

Reply:
I have never heard of flippable the adjective, but the meaning is fairly clear to me. Would it not make it more clear by reformulating slightly, rather than just repeat "flippable element", e.g.
"No flippable element selected:\nPlease select an element that can be flipped and retry"

Reply:
That sounds good. Thanks!